projects/1.18/assets/tiny-pipes/tinypipes/lang/zh_cn.json

```
@@ -0,0 +1,24 @@
+{
+  "itemGroup.tinypipes": "迷你管道",
+  "item.tinypipes.item_pipe": "迷你物品管道",
+  "message.item.tinypipes.item_pipe": "可以在容器之间传输物品。\n手持红石扳手右击改变输入输出面。\n空手右击以打开GUI。",
+  "item.tinypipes.item_filter_pipe": "迷你物品过滤管",
+  "message.item.tinypipes.item_filter_pipe": "可以在容器之间传输被允许的物品。\n空手右击以打开过滤器。\n手持红石扳手右击改变输入输出面。",
+  "item.tinypipes.fluid_pipe": "迷你流体管道",
+  "message.item.tinypipes.fluid_pipe": "可以在流体储罐之间传输流体。\n手持红石扳手右击改变输入输出面。\n空手右击以打开GUI。",
+  "item.tinypipes.fluid_filter_pipe": "迷你流体过滤管道",
+  "message.item.tinypipes.fluid_filter_pipe": "可以在流体储罐之间传输被允许的流体。\n空手右击可使用桶装液体设置过滤器。\n手持红石扳手右击改变输入输出面。",
+  "item.tinypipes.energy_pipe": "迷你能量管道",
+  "message.item.tinypipes.energy_pipe": "可以把FE从输入端传输到接收端。\n手持红石扳手右击改变输入输出面。\n空手右击可使用流体桶设置过滤器。",
+  "item.tinypipes.redstone_pipe": "迷你红石能量管道",
+  "message.item.tinypipes.redstone_pipe": "可以将多个红石信号无损传输。\n手持染料右击连接处来改变颜色频道。\n将只输出至颜色一致的端口处。\n手持红石扳手右击改变输入输出面。\n空手右击以打开GUI。",
+  "tinypipes:item_filter": "物品过滤名单",
+  "tinypipes:fluid_filter": "液体过滤名单",

```

这两句不知道游戏内是体现为一个名单还是和一般的过滤器一样，如果一样，那应该是
```suggestion
  "tinypipes:item_filter": "物品过滤器",
  "tinypipes:fluid_filter": "液体过滤器",
```
以此保证前后文统一